index.html

```diff
@@ -1,13 +1,23 @@
 <!DOCTYPE HTML>
 
 <html>
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
`font-awesome` is already in the `master` branch and should be inside the `<head>` tag. 
Can you remove this